### PR TITLE
new icon: envoy (original, original-wordmark, plain, plain-wordmark)

### PR DESCRIPTION
**Pull request by kilian-paquier**
## Double check these details before you open a PR

- [x] PR does not match another non-stale PR currently opened
- [x] PR name matches the format *new icon: <i>Icon name</i> (<i>versions separated by comma</i>)*. More details [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#overview)
- [x] PR's base is the `develop` branch.
- [x] Your icons are inside a folder as seen [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#organizational-guidelines)
- [x] SVG matches the standards laid out [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#svgStandards)
- [x] A new object is added in the `devicon.json` file as seen [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#-updating-the-deviconjson-)

**This PR closes NONE**

## Link to prove your SVG is correct and up-to-date.

https://www.envoyproxy.io/
https://www.envoyproxy.io/docs/envoy/latest/start/start

### CODE REVIEW

**Review by Panquesito7 — APPROVED** [review]
Hey there! Could you please resolve the merge conflicts?
Let us know if you need any help with this! 🙂


__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix&nbsp; | &nbsp;**Tags:** Inefficient

**Comment by kilian-paquier** [comment]
Rebase done 🙂

File: devicon.json [hunk 1]
```diff
@@ -7309,6 +7309,28 @@
         "color": "#903a36",
         "aliases": []
     },
+    {
+        "name": "envoyproxy",
+        "altnames": [
+            "envoy"
+        ],
```
Comment (Snailedlt) [inline devicon.json:7316]: imo these should switch places. The website calls it envoy, and the logo says envoy, so I would think the icon name should be envoy too
```suggestion
        "name": "envoy",
        "altnames": [
            "envoyproxy"
        ],
```

File: devicon.json [hunk 2]
```diff
@@ -7309,6 +7309,28 @@
         "color": "#903a36",
         "aliases": []
     },
+    {
```
Comment (Snailedlt) [inline devicon.json:7312]: Remember to move this so it fits alphabetically with the rest of the icons

File: devicon.json [hunk 3]
```diff
@@ -7309,6 +7309,28 @@
...
+        "name": "envoyproxy",
+        "altnames": [
+            "envoy"
+        ],
+        "tags": [
+            "middle proxy",
```
Comment (Snailedlt) [inline devicon.json:7318]: Never heard of middle proxy before. Did you mean `proxy`?
```suggestion
            "proxy",
```

**Review by Snailedlt — CHANGES_REQUESTED** [review]
Imo the icon should be renamed to envoy, since the website and logo calls it that. Please tell me if I'm wrong though.

Other than that, there's just one comment on a tag.

**PS:** When renaming the icon, remember to rename the files, folders and PR title too

**Comment by Snailedlt** [comment]
@Panquesito7 one of the checks is failing because the PR title was changed after the PR was made. I've re-ran the job, but it didn't work. Any ideas on how to fix it?
image

image

**Comment by kilian-paquier** [comment]
> @Panquesito7 one of the checks is failing because the PR title was changed after the PR was made. I've re-ran the job, but it didn't work. Any ideas on how to fix it? image
> 
> image

The first commit needed to be renamed too

**Review by Snailedlt — APPROVED** [review]
LGTM! :heavy-checkmark:

**Review by Panquesito7 — APPROVED** [review]
Nice! Thank you. 🚀


__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :trophy: This code is awesome&nbsp; | &nbsp;**Tags:** Efficient, Maintainable

**Pull request merged**